CONTRIBUTING.rst

```diff
@@ -83,9 +83,52 @@ Using a custom ``virtualenv``
     $ cd ${HOME}/hack-on-google-cloud-python
     $ ${VENV}/bin/python setup.py install
 
+- To keep your ``virtualenv`` from getting checked-in, you can make a
+  global gitignore file::
+
+  $ echo "${ENV_NAME}" >> ~/gitignore_global
+  $ git config --global core.excludesfile ~/.gitignore_global
```

Comment:
I don't think this is necessary. This teaches people how to avoid making `git` mistakes, but that's not our concern.